e2e-tests/cypress/e2e/say-at-pre-application-stage.feature

```diff
@@ -7,15 +7,15 @@ Feature: Having your say at the pre-application stage page
     Background: Navigate to page
         Given I navigate to Having your say at the pre-application stage page
 
-    Scenario: verify page title and click Nationally Significant Infrastructure planning process step by step link
+    Scenario: verify page title and click NHaving your say about a national infrastructure project link
```

Comment:
Typo in NHaving

Reply:
Darn, checked and double checked it - will fix now